src/redux/constants/_initialState.js

```diff
@@ -4,4 +4,118 @@ export const initialState = {
     isOpen: false,
     type: "",
   },
+  postings: [
+    {
+      id: 0,
+      title: "Game Boy Advanced SP",
+      datePosted: "2018-07-01T00:00:00.000Z",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu est volutpat, luctus mauris id, bibendum sem. Mauris ultrices diam eu metus mattis, ut sodales tellus rhoncus. Nunc eleifend rutrum tortor, sit amet mattis mi euismod ac. Mauris diam ante, tincidunt in mauris et, ullamcorper fermentum urna.",
+      category: "electronics",
+      condition: "good",
+      quantity: 1,
+      tags: ["kid friendly", "retro gaming"],
```

Comment:
Might need to revisit tags depending on whether we decide to allow the users to create tags or if we decide to have the users choose from a set of pre-determined tags.